cost/aws/reserved_instances/recommendations/aws_reserved_instance_recommendations.pt

```diff
@@ -458,7 +458,7 @@ script "js_currency", type:"javascript" do
   result = ds_currency_target
   result['exchange_rate'] = 1
 
-  if (ds_currency_conversion[0]['to'] != undefined) {
+  if (ds_currency_conversion.length > 0 && ds_currency_conversion[0]['to'] != undefined) {
```

Comment:
I think the ds_currency_conversion[0]['to'] section should be removed or else it's still going to generate the error.